Gopkg.toml

```diff
@@ -8,23 +8,23 @@
 
 [[override]]
   name = "k8s.io/api"
-  version = "kubernetes-1.11.2"
+  version = "kubernetes-1.11.3"
```

Comment:
Should we go ahead and bump up to 1.11.5?

Also, should we update the `pkg/scaffold/gopkgtoml.go` with these same deps?

Reply:
No opposition here to bumping to 1.11.5. And yes good catch I will update the scaffold as well.